tests/DownstreamRestApiTests/AcquireTokenResultTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.Identity.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace UnitTests
```

Comment:
If we use different namespaces for different tests, the test explorer will be easier to navigate.